### ISSUE TITLE
Can't resize tab

### ISSUE THREAD

**Issue by Tisten** [OurMachinery/themachinery-public#1409]
- **Version:** Git Master Branch
- **OS:** Windows

I started the machinery and resized the application and the tabs a bit. The I created a new project which made the tabs change size again so I wanted to change them back. Suddenly there is no way to find the border between Entity Tree and Scene tabs.
![Resize](https://user-images.githubusercontent.com/6726075/174835326-98f536c4-c0f9-467b-be61-398307b04ea3.gif)

**Comment by niklas-ourmachinery** [comment]
Do you have a repro case for it.

If you close the project and re-open it, is the issue still there? In that case, could you share the settings file for the project so we can investigate what the tab layout is that is causing this?

**Comment by Tisten** [comment]
The issue dissapeared when I made the whole window bigger, then the border became selectable again.

And yes, I was able to reproduce it again now, and much simpler. I just made the size of the window smaller and the border wasn't selectable. I saved a project and closed the editor, and then started it again with that project, and the problem was the same.

According to my project settings, the window size this time was:
```
                        "window_width": 1209,
                        "window_height": 804,
```
And I'm using 150% scale on that screen. If I drag it to my other screen which has scale 100%, the problem persists. Now the project settings say
```
                        "window_width": 806,
                        "window_height": 536,
```
